$OpenBSD: patch-src_roff_troff_input_cpp,v 1.1 2011/03/19 16:48:53 schwarze Exp $
# Avoid a name clash over "input_iterator" and let this build on GCC 2.
--- src/roff/troff/input.cpp.orig	Fri Dec 31 00:33:08 2010
+++ src/roff/troff/input.cpp	Thu Mar 17 16:47:28 2011
@@ -156,6 +156,7 @@ static symbol get_delim_name();
 static void init_registers();
 static void trapping_blank_line();
 
+#define input_iterator my_input_iterator
 class input_iterator;
 input_iterator *make_temp_iterator(const char *);
 const char *input_char_description(int);
